chart/k8gb/templates/coredns-tcp-svc.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.coredns.deployment.enabled (not .Values.route53.enabled) }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+{{ include "chart.labels" . | indent 4  }}
+  name: {{ .Release.Name }}-coredns-tcp
+spec:
+  type: NodePort
+  selector:
+    app.kubernetes.io/instance: k8gb
+    app.kubernetes.io/name: coredns
+  ports:
+  - name: tcp-5353
+    port: 53
+    protocol: TCP
```

Comment:
Was it tested outside of the local scenario? Main concern is overall TCP support/expectation in the DNS environment

Reply:
@jkremser could you please move this service definition out of helm and deploy it during test setup creation (around we deploy edgeDNS)

Reply:
it should be there